tests/conftest.py

```diff
@@ -14,7 +14,8 @@ def vm_logger():
 
     # level = TRACE_LEVEL_NUM
     # level = logging.DEBUG
-    level = logging.INFO
+    # level = logging.INFO
+    level = logging.ERROR
```

Comment:
FYI: This came from me ending up with a terminal full of `INFO` level logging messages long enough to push the actual test failure off the history of my terminal.